code/style.css

```diff
@@ -1,4 +1,233 @@
+  text-shadow: 5px 5px rgb(194, 194, 194);
+  min-height: 4rem;
+}
+
+a {
+  color: black;
+  text-decoration: none;
+}
+
+/*Covid virus logo*/
+.logo-imgs {
+  display: none;
+  height: 6rem;
+}
+
+.logo {
+  height: 6rem;
+}
+
+.logo1 {
+  position: relative;
+}
+
+.logo2 {
+  position: absolute;
+  opacity: 0;
+}
+
+.logo2:hover {
+  animation: fadein 1s ease-in alternate infinite;
+}
+
+@keyframes fadein {
+  from { opacity: 0; }
+  to { opacity: 1; }
+}
+
+/*links to soc-med*/
+.soc-med {
+  display: none;  
+  grid-template-columns: repeat(4, 1fr);
+  align-items: center;
+  gap: 2rem;
+}
+
+.soc-icon:hover,
+.nav-links:hover,
+.xpand-button:hover {
+  color: rgb(167, 15, 15);
+}
+
+/* News ticker */
+.black-stripe {
+  background-color: #000000;
+  height: 1.5em;
+  color: white;
+  font-weight: 900;
+  line-height: 100%;
+  text-align: right;
+}
+
+/* Navbar with links to categories*/
+.navbar {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.nav-links {
+  font-size: min(2rem, 4vw);
+  padding: 0.5rem;
+  max-width: max-content;
+  font-weight: 500;
+}
+
+h1 {
+  font-size: 4rem;
+  text-align: center;
+  text-decoration: overline underline;
+  margin: 1rem;
+  padding: 2rem;
+}
+
+.big-news {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-items: center;
+  margin: 1rem 0;
+  gap: 20px;
+}
+
+.big-pic {
+  max-height: 90vw;
+}
+
+.big-news-text {
+  text-align: center;
+  padding: 10px;
+  font-size: max(1.3rem, 2vw);
+}
+
+.cards {
+  background: rgb(240, 239, 239);
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.card {
+  background: rgb(253, 253, 253);
+  margin: 1rem;
+  max-width: min(100vw, 400px);
+  max-height: 25rem;
+  box-shadow: 0px -2px 20px 2px rgba(0, 0, 0, 0.4);
+  transition: 400ms;
+}
+
+/* first row of the card with category and 'more' button*/
+.category {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 1rem;
+}
+
+.xpand-button {
+  color: rgb(104, 104, 104);
+  font-size: 1rem;
+  background-color: white;
+  border: none;
+}
+
+/*Second row of cards with pic*/
+.card-pic {
+  max-width: calc(100vw - 2rem);
+}
+
+/*Third row of card with title of article*/
+.card-title {
+  margin-top: 0.5rem;
+  padding: 0 1rem;
+  text-align: center;
+}
+
+/*Fourth row of card with article content
+  hidden before 'more' button is clicked*/
+.content {
+  padding: 1rem;
+  margin: 0;
+  
+  font-family: "Georgia";
+  line-height: 1.5rem;
+}
+
+.hide {
+  display: none;
+}
+
+.expand {
+  max-height: 40rem;
+}
+
+footer {
+  margin-top: 5rem;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  align-items: center;
+}
+
+/* Changes header to flex container for larger screens 
+    makes virus logo and soc-med links visible on large screens*/
+@media (min-width: 1025px) {
+  header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .big-news {
+    grid-template-columns: 2fr 3fr;
```

Comment:
Like you mentioned, 3 cards instead of 4, but I think it is a good initiative to go with the more stylish option!